package.json

```diff
@@ -303,9 +326,12 @@
         "semver": "^5.6.0",
         "sleep-promise": "^8.0.1",
         "tcp-port-used": "^1.0.1",
+        "triple-beam": "^1.3.0",
```

Comment:
I thought this was removed, but it looks like it made its way back in. Will delete.